mongoConnection.ts

```diff
@@ -0,0 +1,19 @@
+import { MongoClient } from "mongodb";
+import * as dotenv from "dotenv";
+
+// was having an issue with the types, we can fix this later...
+let _connection: any = undefined;
+let _db: any = undefined;
+
+export async function dbConnection() {
+	dotenv.config();
```

Comment:
dotenv.config() should only be called once in an app, so we should call this in our root server file at the highest level possible, not in any individual function.

Reply:
Gotcha, that makes sense!